Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/RotationManipulators.cpp

```diff
@@ -13,45 +13,74 @@
 namespace AzToolsFramework
 {
     RotationManipulators::RotationManipulators(const AZ::Transform& worldFromLocal)
+        : m_viewAngularManipulator { AngularManipulator::MakeShared(worldFromLocal) }
     {
+        const auto saveMostRecentActionCb = [this](const AngularManipulator::Action& action)
```

Comment:
nit: This is very minor but in manipulator code (AzToolsFramework) we tend to use `Fn` postfix for lambda variables as opposed to `Cb` but it's not a huge deal, just making you aware 🙂 (can change or not, not blocking)

Reply:
Ok will do :)